src/ureq_dns_resolver.rs

```diff
@@ -1,19 +1,13 @@
 use std::sync::Arc;
-
-use rustls::ServerName;
 use ureq::Agent;
-use webpki::TlsServerTrustAnchors;
 
-use self::certificate_verifier_no_hostname::CertificateVerifierNoHostname;
-/*
-   The following modules `fixed_resolver` and `certificate_verifier_no_hostname` are from this
-   website:
-   https://scvalex.net/posts/48/
-*/
 mod fixed_resolver {
+
     //! A DNS resolver that always returns the same value regardless of
     //! the host it was queried for.
-
+    //!
+    //! NB: The following modules `fixed_resolver` is from this website:
+    //! https://scvalex.net/posts/48/
     use std::net::SocketAddr;
 
     pub struct FixedResolver(pub SocketAddr);
```

Comment:
I don't see the usage of the URL anywhere. Where is it used?

Reply:
The URL is used to make the post req: https://github.com/mithril-security/blindai/blob/telemetry/src/telemetry.rs#L163